x-pack/filebeat/module/cylance/protect/test/generated.log-expected.json

```diff
@@ -1353,7 +1353,7 @@
         "user.name": "imveni"
     },
     {
-        "@timestamp": "2019-07-25T11:47:41.000Z",
+        "@timestamp": "2020-07-25T11:47:41.000Z",
```

Comment:
these are going to be stripped next time they're run with a `GENERATE` flag and result in a diff, do we just want to strip them out now? i.e.:

```diff
--- a/x-pack/filebeat/module/cylance/protect/test/generated.log-expected.json
+++ b/x-pack/filebeat/module/cylance/protect/test/generated.log-expected.json
@@ -1,6 +1,5 @@
 [
     {
-        "@timestamp": "2016-01-29T08:09:59.000Z",
         "event.action": "ZoneAdd",
         "event.code": "CylancePROTECT",
         "event.dataset": "cylance.protect",
@@ -26,7 +25,6 @@
         "rsa.network.alias_host": [
             "nostrud4819.mail.test"
         ],
-        "rsa.time.event_time": "2016-01-29T08:09:59.000Z",
         "service.type": "cylance",
         "tags": [
             "cylance.protect",
@@ -34,7 +32,6 @@
         ]
     },
```

Reply:
I ran generate again and they are all removed now. Good call.